keysign/discover.py

```diff
@@ -42,18 +42,20 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
-            msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
-            key_data, success, message = msg_tuple
-            if key_data:
+            if BluetoothReceive is not None:
+                self.bt = BluetoothReceive(self.bt_port)
+                msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
+                key_data, success, message = msg_tuple
+                if key_data:
                 # If we found the key
+                    returnValue((key_data, success, message))
+            else:
+                log.info("PyBluez is probably missing.")
+                key_data = None
```

Comment:
Leave this part outside the `else`, otherwise if we don't have a `bt_code` and we only have a `key_data` that fails, we will never enter in this bt block and we miss the return statement.